product_docs/docs/pgd/5/manual/08-using-pgd-cli.mdx

```diff
@@ -0,0 +1,252 @@
+* Swith write leader with the `switchover` command
+
+We go into more details of these command in the worked example below.
+
+## Worked example
+
+### Creating a configuration file
+
+The PGD CLI configuration file is similar to the PGD proxy configuration filer.
+It is a YAML file which contains a cluster object. This has two properties:
+
+The name of the PGD cluster's top-level group (as `name`).
+An array of endpoints of databases (as `endpoints`).
+
+```
+cluster:
+  name: pgd
+  endpoints:
+    - host=host-one dbname=bdrdb port=5444
+    - host=host-two dbname=bdrdb port=5444
+    - host=host-three dbname=bdrdb port=5444
+```
+
+Note that the endpoints in this example specify port=5444.
+This is necessary for EDB Postgres Advanced Server instances.
+For EDB Postgres Extended and Community PostgreSQL, this can be omitted.
+
+### Installing the PGD CLI configuration
+
+Create the PGD CLI configuration directory.
+
+```
+sudo mkdir -p /etc/edb/pgd-cli
+```
+
+Then write the configuration to the `pgd-cli-config.yml` file in the `/etc/edb/pgd-cli` directory.
+
+For our example, this could be run on host-one to create the file.
+
+```
+cat <<EOF | sudo tee /etc/edb/pgd-cli/pgd-cli-config.yml
+cluster:
+  name: pgd
+  endpoints:
+    - host=host-one dbname=bdrdb port=5444
+    - host=host-two dbname=bdrdb port=5444
+    - host=host-three dbname=bdrdb port=5444
+EOF
+```
+
+### Run PGD CLI
+
+With the configuration file in place, we can run pgd-cli. For example, we can use the `show-nodes` command to list the nodes in our cluster and their status.
+
+```
+pgd show-nodes
+__OUTPUT__
+Node       Node ID    Group Type Current State Target State Status Seq ID
+----       -------    ----- ---- ------------- ------------ ------ ------
+node-one   2824718320 dc1   data ACTIVE        ACTIVE       Up     1
+node-three 1954860017 dc1   data ACTIVE        ACTIVE       Up     3
+node-two   2299992455 dc1   data ACTIVE        ACTIVE       Up     2
+```
+
+### Run PGD CLI without a configuration file
+
+You can pass the connection string directly to PGD CLI, using the `--dsn` flag.
+
+```
+pgd --dsn "host=node-one dbname=bdrdb port=5444" show-nodes
+__OUTPUT__
+Node       Node ID    Group Type Current State Target State Status Seq ID
+----       -------    ----- ---- ------------- ------------ ------ ------
+node-one   2824718320 dc1   data ACTIVE        ACTIVE       Up     1
+node-three 1954860017 dc1   data ACTIVE        ACTIVE       Up     3
+node-two   2299992455 dc1   data ACTIVE        ACTIVE       Up     2
+```
+
+### Run PGD CLI with an alternative configuration file
+
+You can create a yaml file with the configuration data for PGD CLI and use that instead of the default configuration by using the `-f` flag of PGD CLI and passing the filename. If our alternative configuration is stored in `$HOME/otherconfig.yml` then running:
+
+```
+pgd -f $HOME/otherconfig.yml show-nodes
+```
+
+## Using PGD CLI to explore the cluster
```

Comment:
There's nothing wrong with this section, but it duplicates doc already covered here: https://deploy-preview-5104--edb-docs-staging.netlify.app/docs/pgd/latest/cli/command_ref/
At a minimum it should link to the full-coverage section, or the entire section could just point over there.

Reply:
Left as a follow through on the instructions.